README.md

```diff
@@ -80,6 +87,19 @@ Here, `MyLauncher` has three options as implemented in `command_launchers.py`: l
 
 Once all jobs have reached either a successful or failed state, you can proceed to remove the records associated with the failed jobs using the command ``python -m domainrobust.scripts.sweep delete_incomplete``, so that the folders associated with the incomplete jobs can be deleted, otherwise other sweep will not launch the job with the same hyperparameters. After deleting the incomplete jobs, you can re-launch them by executing ``python -m domainrobust.scripts.sweep launch``. Please ensure that you provide the same command-line arguments when you re-launch as you did during the initial launch.
 
+For specific algorithms (e.g. SROUDA, DART) that requires pretrained models, it can be generated via algorithm DANN:
```

Comment:
Since this is required before running the actual algorithms, can you please move this to before line 42 ("To train a single model..."). I'd also suggest clarifying this, e.g., "We first generate a pretrained model using DANN (this pretrained model will be used by algorithms like SROUDA and DART)"